packages/core/src/generators/react/types.ts

```diff
@@ -15,4 +15,5 @@ export interface ToReactOptions extends BaseTranspilerOptions {
    */
   contextType?: 'context' | 'prop-drill';
   addUseClientDirectiveIfNeeded?: boolean;
+  noPrefixSlots?: boolean;
```

Comment:
Given https://github.com/BuilderIO/mitosis/issues/1301, I think we want the default to be not prefixing.

Would you mind switching this to: 

```suggestion
  prefixSlots?: boolean;
```

and only prefixing if the option was explicitly provided?


This will require updating the snapshot tests, which you can do by running:

- `yarn ci:build` in the root, and then
- `yarn snapupdate` in `package/core`

Reply:
Scratch that: if you don't mind, completely remove the prefixing logic altogether! We don't need it.

Reply:
👍